test/functional/projects_controller_test.rb

```diff
@@ -241,8 +241,7 @@ class ProjectsControllerTest < ActionController::TestCase
                       '21_unit' => parameters['21_unit'], '22_name' => parameters['22_name'], '22_unit' => parameters['22_unit'],
                       "#{new_lat_field_id}_name" => @project.fields[num_fields - 2].name, "#{new_lat_field_id}_unit" => @project.fields[num_fields - 2].unit,
                       "#{new_long_field_id}_name" => @project.fields[num_fields - 1].name, "#{new_long_field_id}_unit" => @project.fields[num_fields - 1].unit,
-                      '20_restrictions' => 'a,b,c',
-                      '25_name' => 'Location of Foodz', '25_unit' => '', new_field: '' }, user_id: @kate.id
+                      '20_restrictions' => 'a,b,c' }, user_id: @kate.id
```

Comment:
'25_name' => 'Location of Foodz', '25_unit' => '', new_field: '' "

This line was making the test fail. I don't understand why it's there. This test is creating that field and that line does nothing to change the field. Any thoughts?